### DISCUSSION TITLE
Model distribution by user

### DISCUSSION THREAD

**Discussion by mazierovictor** [open-webui/open-webui#6310] [General]
Hello everyone, how are you?!

How can I distribute the models per user, for example:

Model 1 = users A and B

Model 2 = users C and D

And then users do not have access to those models that are not authorized.

**Comment by silenceroom** [comment]
this can be done from the user management function.
1. define the user group.
2. on the model list, selct the model as private and then you can selsect which group can access the available model.

**Comment by GylanSalih** [comment]
To distribute models per user, you can set up access control by defining user roles and linking them to specific models, ensuring only authorized users can access the models assigned to them. Implementing a database that maps users to models and adding backend logic to enforce these permissions will control access based on user authorization.
